### DISCUSSION TITLE
How to run Fix-OneDrive-Zip?

### DISCUSSION THREAD

**Discussion by carmenquezada** [pmqs/Fix-OneDrive-Zip#14] [Q&A]
Of course! Here are the adjustments I made:
Hi! I'm trying to run code to open large OneDrive zipped files, but I need help understanding how to do it. I'm a Mac user, and I tried writing the following command in Terminal:
`perl fix-onedrive-zip [--dry-run] /Users/carmen/Desktop/Second\ Year/Research\ ideas/Databases/Attendance_daily/2015/2015.zip`
where, 
`/Users/carmen/Desktop/Second\ Year/Research\ ideas/Databases/Attendance_daily/2015/2015.zip` is the path of the file I need to unzip. However, I get the following error: "Can't open perl script 'fix-onedrive-zip': No such file or directory." I think I need to download something, but I don't know what or where.

Thank you so much for your help!

**Comment by pmqs** [comment]
drop the `[--dry-run]` part

Try running this

```
perl fix-onedrive-zip /Users/carmen/Desktop/Second\ Year/Research\ ideas/Databases/Attendance_daily/2015/2015.zip
```

**Comment by carmenquezada** [comment]
Thank you so much!

**Comment by pmqs** [comment]
Excellent!
